sdk/python/feast/permissions/auth_model.py

```diff
@@ -11,9 +11,12 @@ class OidcAuthConfig(AuthConfig):
     auth_discovery_url: str
```

Comment:
rename  `OidcAuthConfig(AuthConfig)` ->  `OidcServerAuthConfig(AuthConfig)` WDYT?

Reply:
Client is inheriting this class so it may not be appropriate. In case if some other parameters needs then we can add separate class for server.